src/installer/pkg/projects/netcoreapp/pkg/workaround/Microsoft.NETCore.App.pkgproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Dependency Include="Microsoft.NETCore.Platforms" Version="$(Version)" />
+    <Dependency Include="Microsoft.NETCore.Platforms" Version="5.0.0" />
```

Comment:
Why not just use `Version`?

Reply:
This is an non-shipping package and Version=5.0.0-dev

Reply:
Ah I see... makes sense. How did we do that in core-setup? Did we introduce a variable for Platforms and Targets packages? To avoid duplicating this in a lot of places?

Reply:
> How did we do that in core-setup? 

we flowed through dependencies. https://github.com/dotnet/core-setup/blob/release/3.1/eng/Versions.props#L60

> Did we introduce a variable for Platforms and Targets packages? To avoid duplicating this in a lot of places?

i can add a new one for this